acoustid.py

```diff
@@ -348,6 +353,55 @@ def fingerprint_file(path, maxlength=MAX_AUDIO_LENGTH, force_fpcalc=False):
         return _fingerprint_file_fpcalc(path, maxlength)
 
 
+
+
+
+
+
+def compare_fingerprints(a, b) -> float:
+    """
+    compare two fingerprints locally
+    :param a: fingerprint of acoustid.fingerprint_file(filepath_a)
+    :param b: second fingerprint of acoustid.fingerprint_file(filepath_b)
+    :return:  similarity score [0,1]
+    """
+
+    if not have_chromaprint:
+        raise ModuleNotFoundError("function needs chromaprint")
+
+    # decompress fingerprints
+    a = chromaprint.decode_fingerprint(a)[0]
+    b = chromaprint.decode_fingerprint(b)[0]
+    a = list(map(int, a))
+    b = list(map(int, b))
+
+    ACOUSTID_MAX_BIT_ERROR = 2
+    ACOUSTID_MAX_ALIGN_OFFSET = 120
```

Comment:
Perhaps these should be module-level constants (at the top of the Python file).

Reply:
Also, it's probably OK to call these just `MAX_BIT_ERROR` and `MAX_ALIGN_OFFSET` because we know we're dealing with acoustid inside this module.